pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -655,14 +657,16 @@ def rebase_pass(self) -> BasePass:
             allow_swaps=self.compilation_config.allow_implicit_swaps,
         )
 
-    def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
+    def default_compilation_pass(
+        self, optimisation_level: int = 2, timeout: int = 300
```

Comment:
It seems the user cannot set a `timeout` when calling `get_compiled_circuit`. Any thoughts on this?

Reply:
I've added `get_compiled_circuit` and `get_compiled_circuits` to `QuantinuumBackend` with a timeout parameter